utils/answer-types.js

```diff
@@ -1232,7 +1233,6 @@ Khan.answerTypes = $.extend(Khan.answerTypes, {
                         guess: guess
                     };
                 } else {
-                    score.empty = false;
```

Comment:
Checking my understanding... Is this the big killer that was keeping everything from being marked empty?